spec/controllers/users_controller_spec.rb

```diff
@@ -83,11 +83,15 @@
   end
 
   context "DELETE #destroy" do
-    context "when not logged in" do
-      it "won't delete the user" do
-        delete :destroy, params: { id: 1 }
+    context "user doesn't exist" do
```

Comment:
First of all: this method does not belong in a PR about views

Then a question is: do we want a user to actually be able to delete their user record?
If we do, I would check that in a feature specs that would be as follows:
1. user logs in
2. user deletes their user record
3. user gets logged out once deletion is successful

Reply:
Sorry, agreed on both accounts, I have deleted the spec as it should be an admin deletes a user account. Sha: bb55f12